components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/BuildScanDataFields.java

```diff
@@ -77,6 +77,6 @@ private static String summaryTotal(BuildScanData data, String avoidanceOutcome,
     }
 
     private static String formatDuration(Duration duration) {
-        return duration == null ? NO_VALUE : String.valueOf(duration.toMillis());
+        return duration == null ? NO_VALUE : DurationFormat.format(duration);
```

Comment:
Small detail: I would use a static import here, too, just like you do it on BuildTimeMetricsFields.

Reply:
I like it. 👍 Done in 44624dad3bd2a29b0741115eab53a80700cf4b75